test/integration/testdata/egress-proxy.yaml.tmpl

```diff
@@ -35,7 +35,17 @@ spec:
         imagePullPolicy: IfNotPresent
         ports:
         - containerPort: 80
-        args: ["proxy", "egress", "-v", "{{.Verbosity}}"]
+        args:
+        - proxy
+        - egress
+        - -v
+        - "{{.Verbosity}}"
+        - --config
+        - |
```

Comment:
Not familiar with this - does it generate a temp file ?

Reply:
Yes, as I mentioned this is how it looks when generated by kube-inject or admission controller. Ingress is really a dummy pod injected with a proxy.